Program.cs

```diff
@@ -1,18 +1,43 @@
 ﻿using MongoDB.Driver;
 using MongoDB.Bson;
+using System.Diagnostics;
 
 public static class Example
 {
     public static void Main(string[] args)
     {
-        var client = new MongoClient(args[0].ToString());
+        var connectionUri = args[0].ToString();
+        var settings = MongoClientSettings.FromConnectionString(connectionUri);
+        var client = new MongoClient(settings);
 
         IMongoDatabase db = client.GetDatabase("test");
         var command = new BsonDocument { { "ping", 1 } };
         var res = db.RunCommand<BsonDocument>(command);
+        Console.WriteLine(res);
 
+        command = new BsonDocument { { "dropDatabase", 1 } };
+        res = db.RunCommand<BsonDocument>(command);
         Console.WriteLine(res);
 
+        var documentList = new List<BsonDocument>();
+        for (int i = 1; i < 5; i++)
+        {
+            var document = new BsonDocument{ {"_id", i }, { "a", i } };
+            documentList.Add(document);
+        }
+
+        // insert documents
+        var collection = db.GetCollection<BsonDocument> ("foo");
+        collection.InsertMany(documentList);

```

Comment:
`ping` and `dropDatabase` has response printed with ` Console.WriteLine(res)`, why do we not do that for `insertMany` and `Find`? :thinking:

Reply:
Ah I see :+1: